test/common/protobuf/BUILD

```diff
@@ -20,6 +22,21 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_benchmark_binary(
+    name = "utility_speed_test",
```

Comment:
can you please name the source file and binary target something_benchmark?

My suggestion is: protobuf_util_benchmark.cc